packages/vaex-ml/vaex/ml/metrics.py

```diff
@@ -1,15 +1,89 @@
+import warnings
+
 import numpy as np
 
 import vaex
 
 
+def _ensure_numpy_array(x):
```

Comment:
you should use `vaex.array_types.to_numpy(x)` instead

Reply:
Good point, done!